$OpenBSD$
Remove tests for include files and libraries there are no
configure options to disable support the build may fail or be altered
if added
--- configure.orig	Tue May 22 20:08:45 2012
+++ configure	Sat Nov 15 13:49:21 2014
@@ -273,51 +273,6 @@
 fi
 
 echo "Checking for SVN (libsvn_client-1 libapr-1.so libaprutil-1.so) ..."
-for i in  $LIBDIRS
-do
-    if [ "X" = "X$SVN_PATH" ]; then
-        if [ -f "$i/libsvn_client-1.so" ]&&[ -f "$i/libapr-1.so" ]&&[ -f "$i/libaprutil-1.so" ]; then
-            SVN_PATH="$i"
-	    APR_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$SVN_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libsvn_client*.so* 2> /dev/null | grep libsvn_client.`
-        if [ -n "$TMP_LIB" ]; then
-          SVN_PATH="$i"
-	  APR_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$SVN_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libsvn_client*.dll* 2> /dev/null | grep libsvn_client.`
-        if [ -n "$TMP_LIB" ]; then
-          SVN_PATH="$i"
-	  APR_PATH="$i"
-        fi
-    fi
-done
-for i in $INCDIRS
-do
-    if [ "X" = "X$SVN_IPATH" ]; then
-        if [ -d "$i/subversion-1" ]; then
-            SVN_IPATH="$i/subversion-1"
-        fi
-        if [ -d "$i/svn" ]; then
-            SVN_IPATH="$i/svn"
-        fi
-    fi  
-    if [ "X" = "X$APR_IPATH" ]; then
-        if [ -d "$i/apr-1.0" ]; then
-            APR_IPATH="$i/apr-1.0"
-        fi
-        if [ -d "$i/apr-1" ]; then
-            APR_IPATH="$i/apr-1"
-        fi
-        if [ -d "$i/apr" ]; then
-            APR_IPATH="$i/apr"
-        fi
-    fi  
-done
 
 if [ "X" = "X$SVN_PATH" -o "X" = "X$SVN_IPATH" -o "X" = "X$APR_IPATH" ]; then
     SVN_PATH=""
@@ -339,34 +294,7 @@
 fi
 
 echo "Checking for firebird (libfbclient.so) ..."
-for i in  $LIBDIRS
-do
-    if [ "X" = "X$FIREBIRD_PATH" ]; then
-        if [ -f "$i/libfbclient.so" ]; then
-            FIREBIRD_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$FIREBIRD_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libfbclient.so.* 2> /dev/null | grep libfbclient.`
-        if [ -n "$TMP_LIB" ]; then
-          FIREBIRD_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$FIREBIRD_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libfbclient.dll* 2> /dev/null | grep libfbclient.`
-        if [ -n "$TMP_LIB" ]; then
-          FIREBIRD_PATH="$i"
-        fi
-    fi
-done
-for i in $INCDIRS
-do
-    if [ "X" != "X$FIREBIRD_PATH" ]; then
-        if [ -f "$i/ibase.h" ]; then
-            FIREBIRD_IPATH="$i"
-        fi
-    fi
-done
+
 if [ -n "$FIREBIRD_PATH" -a -n "$FIREBIRD_IPATH" ]; then
     echo "                                       ... found"
 fi
@@ -419,34 +347,6 @@
       echo "                                       ... math.h not found, module Mysql disabled"
 fi
 echo "Checking for AFP (libafpclient.so) ..."
-for i in  $LIBDIRS 
-do
-    if [ "X" = "X$AFP_PATH" ]; then
-        if [ -f "$i/libafpclient.so" ]; then
-            AFP_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$AFP_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libafpclient.so.* 2> /dev/null | grep libafpclient.`
-        if [ -n "$TMP_LIB" ]; then
-          AFP_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$AFP_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libafpclient.dll* 2> /dev/null | grep libafpclient.`
-        if [ -n "$TMP_LIB" ]; then
-          AFP_PATH="$i"
-        fi
-    fi
-done
-for i in $INCDIRS
-do
-    if [ "X" != "X$AFP_PATH" ]; then
-        if [ -f "$i/afpfs-ng/afp.h" ]; then
-            AFP_IPATH="$i/afpfs-ng"
-        fi
-    fi
-done
 if [ -n "$AFP_PATH" -a -n "$AFP_IPATH" ]; then
     echo "                                   ... found"
 fi
@@ -457,34 +357,7 @@
 fi
 
 echo "Checking for NCP (libncp.so / nwcalls.h) ..."
-for i in  $LIBDIRS
-do
-    if [ "X" = "X$NCP_PATH" ]; then
-        if [ -f "$i/libncp.so" ]; then
-            NCP_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$NCP_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libncp.so.* 2> /dev/null | grep ncp.`
-        if [ -n "$TMP_LIB" ]; then
-            NCP_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$NCP_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libncp.dll* 2> /dev/null | grep ncp.`
-        if [ -n "$TMP_LIB" ]; then
-            NCP_PATH="$i"
-        fi
-    fi
-done
-for i in $INCDIRS
-do
-    if [ "X" != "X$NCP_PATH" ]; then
-        if [ -f "$i/ncp/nwcalls.h" ]; then
-            NCP_IPATH="$i"
-        fi
-    fi
-done
+
 if [ -n "$NCP_PATH" -a -n "$NCP_IPATH" ]; then
     echo "                                         ... found"
 fi
@@ -495,28 +368,7 @@
 fi
 
 echo "Checking for SAP/R3 (librfc/saprfc.h) ..."
-for i in  $LIBDIRS 
-do
-    if [ "X" = "X$SAPR3_PATH" ]; then
-        if [ -f "$i/librfc.a" -o -f "$i/librfc.dylib" -o "$i/librfc32.dll" ]; then
-            SAPR3_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$SAPR3_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/librfc.* $i/librfc32.* 2> /dev/null | grep librfc`
-        if [ -n "$TMP_LIB" ]; then
-          SAPR3_PATH="$i"
-        fi
-    fi
-done
-for i in $INCDIRS
-do
-    if [ "X" = "X$SAPR3_IPATH" ]; then
-        if [ -f "$i/saprfc.h" ]; then
-            SAPR3_IPATH="$i"
-        fi
-    fi
-done
+
 if [ "X" = "X$SAPR3_PATH" ]; then
     SAPR3_IPATH=""
 fi
@@ -628,38 +480,6 @@
     LIBDIRS="$LIBDIRS $WORACLE_LIB_PATH"
 fi
 
-for i in  $LIBDIRS
-do
-    if [ "X" = "X$ORACLE_PATH" ]; then
-        if [ -f "$i/libocci.so" ]&&[ -f "$i/libclntsh.so" ]; then
-            ORACLE_PATH="$i"
-        fi
-    fi
-    if [ "X" = "X$ORACLE_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libocci.so.* 2> /dev/null | grep occi.`
-        if [ -n "$TMP_LIB" ]; then
-            ORACLE_PATH="$i"
-        fi
-        if [ "X" != "X$ORACLE_PATH" ]; then
-          TMP_LIB=`/bin/ls $i/libclntsh.so.* 2> /dev/null | grep clntsh.`
-          if [ -z "$TMP_LIB" ]; then
-            ORACLE_PATH=""
-          fi
-        fi
-    fi
-    if [ "X" = "X$ORACLE_PATH" ]; then
-        TMP_LIB=`/bin/ls $i/libocci.dll* 2> /dev/null | grep occi.`
-        if [ -n "$TMP_LIB" ]; then
-            ORACLE_PATH="$i"
-        fi
-        if [ "X" != "X$ORACLE_PATH" ]; then
-          TMP_LIB=`/bin/ls $i/libclntsh.dll* 2> /dev/null | grep clntsh.`
-          if [ -z "$TMP_LIB" ]; then
-            ORACLE_PATH=""
-          fi
-        fi
-    fi
-done
 #check for Kernel Asynchronous I/O (AIO) lib support
 if [ "X" != "X$ORACLE_PATH" ]; then
   LIBAIO="X"
@@ -753,7 +573,7 @@
 
 echo
 XDEFINES=""
-XLIBS=""
+XLIBS="-lm"
 XLIBPATHS=""
 XIPATHS=""
 
